### DISCUSSION TITLE
Quartz extension "Preview" status

### DISCUSSION THREAD

**Discussion by dominic-harris** [quarkusio/quarkus#44228] [Community]
Hi
We would like to use the Quartz extension in our app. as it could make scheduled tasks much easier to handle and more generally make things a lot simpler.

However, the fact that it is currently a 'Preview' feature is a bit of a setback. I have read the FAQ (https://quarkus.io/faq/#what-are-the-extension-statuses). If I understand rightly, as a Preview feature, there is no guarantee that it will be present in future versions.

Are there plans to move Quartz to "Stable" in the near future?

Thanks

**Comment by geoand** [comment]
See https://stackoverflow.com/a/79166715/2504224
